### PR TITLE
Merge future-stabilization into master

### PR DESCRIPTION
**Pull request by vslsnap**
This is an automatically generated pull request from future-stabilization into master.

@dotnet/roslyn-infrastructure:

``` bash
git remote add vslsnap-roslyn "https://github.com/vslsnap/roslyn.git"
git fetch vslsnap-roslyn
git fetch upstream
git checkout merge-future-stabilization-into-master20160609-150148
git reset --hard upstream/master
git merge upstream/future-stabilization
# Fix merge conflicts
git commit
git push vslsnap-roslyn merge-future-stabilization-into-master20160609-150148 --force
```

Once the merge can be made and all the tests pass, you are free to merge the pull request.

### CODE REVIEW

**Comment by vslsnap** [comment]
@dotnet-bot test vsi please

**Comment by jinujoseph** [comment]
@dotnet-bot test windows_eta_open_prtest please 
failure related to https://github.com/dotnet/roslyn/issues/11856

**Comment by jinujoseph** [comment]
@dotnet-bot test vsi please

**Comment by jinujoseph** [comment]
@dotnet-bot retest this please

File: src/Debugging/Microsoft.DiaSymReader.PortablePdb/Microsoft.DiaSymReader.PortablePdb.nuspec [hunk 1]
```diff
@@ -8,9 +8,9 @@
     </description>
     <dependencies>
       <group targetFramework="portable-net45+win8">
-        <dependency id="Microsoft.DiaSymReader" version="1.0.8-rc3-60429-03" />
-        <dependency id="System.Reflection.Metadata" version="1.2.0-rc3-24102-00" />
-        <dependency id="System.Collections.Immutable" version="1.3.0-rc3-24102-00" />
+        <dependency id="Microsoft.DiaSymReader" version="1.0.7" />
```
Comment (jasonmalinowski) [inline src/Debugging/Microsoft.DiaSymReader.PortablePdb/Microsoft.DiaSymReader.PortablePdb.nuspec]: is the rollback here intended?

**Comment by jinujoseph** [comment]
@dotnet-bot test windows_eta_open_prtest 
failure related to #11856

**Comment by jinujoseph** [comment]
@dotnet-bot retest this please

**Comment by jinujoseph** [comment]
@dotnet-bot  test windows_vsi_p3_open_prtest please

**Comment by jinujoseph** [comment]
@dotnet-bot test perf_correctness_prtest  please

File: src/Debugging/Microsoft.DiaSymReader.PortablePdb/Microsoft.DiaSymReader.PortablePdb.nuspec (hunk 1 above)
Comment (tmat) [inline src/Debugging/Microsoft.DiaSymReader.PortablePdb/Microsoft.DiaSymReader.PortablePdb.nuspec]: It doesn't matter. The nuspec is broken in master anyways. I have a change that fixes the versions in master. Waiting for CoreCLR fix to merge it.

Comment (jinujoseph) [inline src/Debugging/Microsoft.DiaSymReader.PortablePdb/Microsoft.DiaSymReader.PortablePdb.nuspec]: So this is godo to be merged in ? or wait for your change to make it in ?

Comment (tmat) [inline src/Debugging/Microsoft.DiaSymReader.PortablePdb/Microsoft.DiaSymReader.PortablePdb.nuspec]: Yes, merge it.

**Pull request merged**